syntax = "proto2";

message update_name_request {
	optional string name = 1;
}